_posts/en/2017-01-11-android-listeners.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: 'Android: Why do I like listeners?'
+lang: en
+permalink: /android-listeners/
+excerpt: "Let's talk about listeners"
+authors:
+    - thuchon
+categories:
+- Mobile
+tags:
+- mobile application
+- mobile
+- dev mobile
+- Android
+---
+### Introduction
+
+Hi Astronauts! Today again, I will keep talking about native mobile development, et this time, it is about Android. As my previous article, this one is for beginners, si it means that we will spend a small amount of time to discover few technical terms together before going deep in the subject. If you did not read my last article Closures VS Delegates, I welcome you to do it, it is a prerequisite in order to understand this one. [Delegates VS Closures](https://blog.eleven-labs.com/en/delegates-closures/){:rel="nofollow noreferrer"}<br />
+For those who don't know, in order to code for Android, you must use Java. So, for the code samples, it will be faster, because only one language to procude, one syntax et one file structure. The important notion in this article is: Listener.
+
+**Listeners**
+
+A listener is a reference to an object that we don't know the exact type, but important thing, it inherits an interface. Because this object inherits an interface, we then know that we can call the methods defined in the interface, even if we don"t know in detail the given object. I think that a proper exemple will help us to understand what I am talking about.<br/>
```

Comment:
of which we don't know the exact type, but the important thing is... implements an interface
don't (you have misspelled don"t)
example